services/management/service_test.go

```diff
@@ -106,6 +106,31 @@ func TestServiceService(t *testing.T) {
 			tests.AssertGRPCError(t, status.New(codes.InvalidArgument, `wrong service type`), err)
 		})
 
+		t.Run("MySQL Conflict socket and address", func(t *testing.T) {
```

Comment:
in other tests we were testing Remove, but here we test adding and it looks weird that we test models package in management package.

Reply:
And what is your suggestion?

Reply:
Extract it from Remove part and maybe remove it since we have api tests

Reply:
Moved to service_helpers_test.go